pkg/controllers/management/auth/token.go

```diff
@@ -3,6 +3,7 @@ package auth
 import (
 	tokenUtil "github.com/rancher/rancher/pkg/auth/tokens"
 	"github.com/rancher/rancher/pkg/features"
+	wrangmgmtv3 "github.com/rancher/rancher/pkg/generated/controllers/management.cattle.io/v3"
 	v3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
```

Comment:
I think this may be able to become v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"